### ISSUE TITLE
Polygon selection

### ISSUE THREAD

**Issue by le-jeu** [wasabee-project/Wasabee-IITC#200]
add polygon selection for portal selection (autodraw, zone outgoing link, markers, colors etc

related: #191

**Comment by TheMitch2** [comment]
Added in 0.20 dev

**Comment by cloudkucooland** [comment]
no, polygon zones was added. this is selecting portals for auto-draw. We can use zones for it, but I'd still like to try something for the autodraws.

**Comment by TheMitch2** [comment]
I missed the distinction. That will be really cool

**Comment by VE6SRV** [comment]
> no, polygon zones was added. this is selecting portals for auto-draw. We can use zones for it, but I'd still like to try something for the autodraws.

Being able to select portals by polygon would be very handy. I have been trying to modify the browser window size to only incorporate specific portals for auto-draw inclusion. There can be situations where physical barriers may make including some portals in an area less that desirable. Using markers to exclude works, but it can take some time to mark all portals for exclusion. 

Polygon selection to be able to add markers to the selected portals would work for this use case, as well as others. Setting portal pin colours is also desirable.

**Comment by uduntuntu** [comment]
Especially starburst / fanfiled planning desire to polygon select option. Homogenous field autodraw would also love the core portals selection and that would be also lighter to calculate.
